test/fixtures/import-malformed.css

```diff
@@ -0,0 +1,35 @@
+.malformed.one:before {
```

Comment:
Simple file to add in some well formed css, and malformed css, so we can easily test against it.